haute-ecole-de-gestion-de-geneve-iso-690.csl

```diff
@@ -17,29 +17,37 @@
     </contributor>
     <category citation-format="author-date"/>
     <category field="generic-base"/>
-    <updated>2023-08-08T00:29:32+00:00</updated>
+    <updated>2023-09-17T00:00:02+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="fr-FR">
     <terms>
       <term name="no date">[sans date]</term>
-      <term name="in">in</term>
       <term name="online">en&#160;ligne</term>
-      <term name="accessed">consulté le</term>
       <term name="retrieved">disponible</term>
-      <term name="from">à l'adresse</term>
-      <term name="translator" form="short">trad.</term>
-      <term name="issue">no</term>
-      <term name="editor" form="short">éd.</term>
+      <term name="issue" form="short">
+        <single>no</single>
+        <multiple>nos</multiple>
+      </term>
       <term name="page" form="short">
         <single>p.</single>
         <multiple>pp.</multiple>
       </term>
+      <term name="number-of-pages" form="short">
+        <single>p.</single>
+        <multiple>pp.</multiple>
+      </term>
+      <!-- the 2 following terms, never used, are hijacked to distinguish the diffrents dates (submitted, issued) of a patent -->
```

Comment:
No, let's not do this  -- completely re-interpreting terms (this is for spring and summer, btw.) doesn't have any benefits over just using `text value` in the style and hardcoding the term, which I'd suggest doing here (since we indeed missed to include the date lables for patents in 1.0.2

Reply:
Hi @adam3smith,

I didn't want to that this way, but...

1. hardcoding the term is not suitable has the HEG citation style is multilingual
2. none of Zotero, Mendeley or Papers have a field corresponding "season-01", "season-02"... CSL variables
3. I really need to find a way to distinguish the submission date from the publication date in the reference

That said, I can wait a little bit. Is there any plan to include date labels for patents in CSL soon?

Reply:
Right, I missed the multilingual part, but you're wrong about the lack of the CSL variables:
Season 1 is just Spring. It's currently not directly accessible via the Zotero date field, but CSL understands ISO extended date forms, so `Issued: 2022-21`  in Extra in Zotero, e.g., will be rendered as the localized season-01 (so Spring 2022 in English, printemps 2022 in French, etc.). A better date field in Zotero can happen at any time, which would allow native input of this. Pandoc, of course, already supports this. In other words, people may already have data in Zotero that would make this appear in the middle of the date in your style.
So if you must 'hijack' a variable, use another term, please.

Reply:
Thanks @adam3smith ! I didn't know that the date could be used as season variable from the `Extra` field.
I understand that updates in CSl cannot be quick.
As it seems that I have no other than hijacking a CSL variable, I'll look for another one.